vme/src/nanny.cpp

```diff
@@ -379,17 +378,18 @@ void pc_data::gstate_togame(dilprg *pdontstop)
     insert_in_unit_list(this);
     unit_to_unit(this, load_room);
 
+    // TODO is this ok coz DILWAY = 1 and !1 is false so unreachable code
     if (CHAR_DESCRIPTOR(this) && !DILWAY) /* Only do these things if player is connected */
     {
-        snprintf(buf, sizeof(buf), "%s has entered the world.<br/>", UNIT_NAME(this));
+        auto msg = diku::format_to_str("%s has entered the world.<br/>", UNIT_NAME(this));
 
         for (i = g_descriptor_list; i; i = i->next)
         {
             if (descriptor_is_playing(i) && i->character != this && CHAR_CAN_SEE(CHAR_ORIGINAL(i->character), this) &&
                 IS_PC(CHAR_ORIGINAL(i->character)) && IS_SET(PC_FLAGS(CHAR_ORIGINAL(i->character)), PC_INFORM) &&
                 !same_surroundings(this, i->character))
             {
-                send_to_descriptor(buf, i);
+                send_to_descriptor(msg, i);
             }
         }
```

Comment:
So this code is unreachable, is that by intention? Line Left-327, Right-329 defines it as 1 so never go inside the `if()`